app/views/providers/means/identify_incomes/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with model: @form, url: providers_legal_aid_application_means_identify_incomes_path(@legal_aid_application), method: :patch do |f| %>
+  <%= page_template page_title: page_title, template: :basic, form: f do %>
+    <%= f.govuk_check_boxes_fieldset :income_types, legend: {size: "xl"} do %>
```

Comment:
```suggestion
    <%= f.govuk_check_boxes_fieldset :income_types, legend: {size: "xl", tag: "h1"} do %>
```

Reply:
Would suggest adding this so the legend renders within a h1 tag - otherwise the page is missing a h1

Reply:
Good spot, thanks!